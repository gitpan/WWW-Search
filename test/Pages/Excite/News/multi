http://search.excite.com/search.gw?c=timely&showSummary=true&perPage=50&start=0&search=prison%2A+AND+escape%2A
http://search.excite.com/search.gw?c=timely&showSummary=true&perPage=50&start=50&search=prison%2A+AND+escape%2A
