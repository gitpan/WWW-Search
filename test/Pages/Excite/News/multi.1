<HTML><HEAD>
<TITLE>Excite Search Results for "prison* AND escape*"</TITLE>
</HEAD>
<BODY bgcolor=ffffff>
<CENTER>
<center>
<table cellspacing=0 cellpadding=0 border=0 width=600>
<tr>
<td align=right VALIGN=BOTTOM width=120>
<a href=http://www.excite.com/><img src=/img/head/logo3.gif width=128 height=50 border=0></a>
</td>
<td width=480>
<table cellspacing=0 cellpadding=0 border=0 width=100%>
<tr>
<td ALIGN=LEFT VALIGN=bottom>
<font size=4 face=helvetica>
<b>Search</b>
</font>
</TD>
<TD> <table cellspacing=0 cellpadding=0 border=0 width=100%>
<TR>
<TD ALIGN=RIGHT VALIGN=BOTTOM NOWRAP>
<form action="/redir.dcg" method=get onChange="submit()">
<td align=right>
<font face="helvetica" size="-1">
<a href="http://www.excite.com/"><b>My Excite</b></a> | <a href="http://mail.excite.com"><b>Check Email</b></a></font>&nbsp;
<select name="url" size="1" onChange="submit()">
<option value=";http://www.excite.com/">Excite Home
<option value=";http://www.excite.com/auctions/">Auctions
<option value=";http://www.excite.com/autos/">Autos
<option value=";http://www.excite.com/careers/">Careers
<option value=";http://www.excite.com/local/">City Guides
<option value=";http://www.excite.com/classifieds/">Classifieds
<option value=";http://www.excite.com/computers_and_internet/">Computers & Internet
<option value=";http://www.excite.com/education/">Education
<option value=";http://www.excite.com/entertainment/">Entertainment
<option value=";http://www.excite.com/family/">Family
<option value=";http://www.excite.com/games/">Games
<option value=";http://www.excite.com/health/">Health
<option value=";http://www.excite.com/home_and_real_estate/">Home &amp; Real Estate
<option value=";http://www.excite.com/horoscopes/">Horoscopes
<option value=";http://www.excite.com/lifestyle/">Lifestyle
<option value=";http://quicken.excite.com/">Money &amp; Investing
<option value=";http://news.excite.com/">News
<option value=";http://www.excite.com/people_and_chat/">People &amp; Chat
<option value=";http://www.excite.com/shopping/">Shopping
<option value=";http://sports.excite.com/">Sports
<option value=";http://city.net/">Travel
</select>
<input type="submit" value="Go!">
</td>
</form>
</TD>
</TR>
</TABLE>
</TD>
</tr>
<tr BGCOLOR=#ff0000>
<td height=2 colspan=2>
<table border=0 cellpadding=1 cellspacing=0 width=100%>
<tr>
<td>
</td>
</tr>
</table>
</td>
</tr>
<tr VALIGN=top>
<td colspan=2>
&nbsp;
</td>
</tr>
</table>
</TD>
</TR>
<TR>
<TD COLSPAN=2>
<FORM NAME=topbox ACTION=/search.gw>
<TABLE border=0 cellpadding=1 cellspacing=0>
<TR><TD bgcolor=#cccccc>
<TABLE border=0 cellspacing=0 cellpadding=4 width='598' bgcolor=ffffff>
<TR>
<TD align=right bgcolor=ffffff>
<b><font face="helvetica" size=-1>You searched for </font></b>
<input size=30 name=search value="prison* AND escape*">
</td>
<td>
<font face="helvetica" size=-1>
<input type=submit value="Search Again">
</font>
</td>
<TD nowrap>
<font face="helvetica" size=-1>
<b><a href=/relocate/co=help;http://www.excite.com/Info/>
Help</a>&nbsp;&nbsp;<br>
<a href=http://www.excite.com/search_forms/advanced/>
More Search</a>
</b></font>
</TD>
</TR>
</TABLE>
<TABLE border=0 cellspacing=0 cellpadding=4 width=598 bgcolor=ffffff>
<TR>
<td align=right colspan=1>
<font face="helvetica" size=-1>
<b>Search:</b>
</font>
</td>
<td align=left colspan=3>
<font face="helvetica" size=-1>&nbsp;&nbsp;
<input type=radio name=c value=hourly ><img src=http://www.excite.com/pfp/excite/images/lep.gif> Newswires&nbsp;&nbsp;
<input type=radio name=c value=timely checked>Web News&nbsp;&nbsp;
<input type=radio name=c value=hourtime >Both	</font>
</td>
</tr>
</table>
<input type=hidden name=showSummary value=true>
<table border=0 cellspacing=0 cellpadding=1 width=100%>
<tr><td align=center bgcolor=ffcc00>
<font face="helvetica" size=-1><b>
Calculate Your: <A HREF=http://www.excite.com/relocate/co=xxsr_htools_iw;http://health.excite.com/calculator/?calc=ideal&sys=us>Ideal Weight</A> | <A HREF=http://www.excite.com/relocate/co=xxsr_htools_bmi;http://health.excite.com/calculator/?calc=bmi&sys=us>Body Mass Index</A> | <A HREF=http://www.excite.com/relocate/co=xxsr_htools_cn;http://health.excite.com/calculator/?calc=calneed&sys=us>Caloric Need</A> | <A HREF=http://www.excite.com/relocate/co=xxsr_htools_fn;http://health.excite.com/calculator/?calc=fatneed&sys=us>Fat Need</A> | <A HREF=http://www.excite.com/relocate/co=xxsr_htools_more;/health/>more</A>
</b></font>
</td></tr>
</table>
</td></tr>
</table>
</form>
</TD>
</TR>
</table>
<form action="http://lifeminders.com/lifeminder30/banner/SignUpDAT.asp"
method="get"> <input type="hidden"
name="MktgSourceCD" value="LLQA58"> <TABLE BORDER=1 CELLPADDING=0
CELLSPACING=0 WIDTH=468 HEIGHT=60><TR><TD>
<table cellpadding="0" cellspacing="0" width="468" height="60"
bgcolor="FF0000" border="0"><tr><td rowspan="2"
valign="top"><img src="http://image.eimg.com/ads/94014_lmhtmlogo.gif" width="96" height="60" border="0"
vspace="0" hspace="0"></td><td height="30"
valign="top" width="372" bgcolor="FF0000"><img src="http://image.eimg.com/ads/94014_contestcopy.gif"
width="370" height="30"
border="0"></td></tr><tr><td valign="top" width="372" height="30"
bgcolor="FF0000"
align="center">&nbsp;&nbsp;<font color="#FFFFFF"><font face="arial,
helvetica, sans-serif" size="-2">Your
E-mail:</font><input type="text" name="Email" size="12" align="top"><font
face="arial, helvetica, sans-serif"
size="-2">&nbsp;&nbsp;Zip Code:</font><input type="text" name="PostalCode"
size="7">&nbsp;&nbsp;&nbsp; <input
type="image" src="http://image.eimg.com/ads/94014_submit.gif" vspace="2" width="50" height="15"
align="bottom"
border="0"></td></tr></table></TD></TR></TABLE><A
HREF="/relocate/ad=94014_htmlad.gif|pk=2920596;http://www.excite.com"></A><CENTER><FONT SIZE=-1
COLOR=#0000FF FACE=arial, geneva><B>Click Here!</B></FONT></CENTER>
</form>
</center>
<small><br></small>
</center>
<a name=news><img src=/img/pixel/trans.gif height=1 width=1><BR></a>
<table border=0 cellpadding=0 cellspacing=0 width=100%><tr>
<td bgcolor=ffcc00 width=25%>
<table border=1 cellpadding=1 cellspacing=0 width=100%><tr>
<td align=center><font face="helvetica" size=-1><b>
<img src=/img/logo/lep_y.gif height=16 width=16>News Articles
</b></font></td></tr></table>
</td><td valign=bottom width=100%>
<table border=0 cellspacing=0 cellpadding=0 width=100%>
<tr><td bgcolor=999999 width=100%><img src=/img/pixel/trans.gif height=2 width=1></td></tr></table>
</td></tr></table>
<font face="helvetica">
<p><b>&nbsp;Web News</b>
(<i><a href="http://nt.excite.com/sources.html"> Sources </a></i>) </font>
<br>
<table border=0 cellpadding=2 cellspacing=0 width=100%>
<tr><td VALIGN=TOP width=40% nowrap>
<font face="helvetica">
Articles
&nbsp;51-62
</font></td>
<td align=right width=60% nowrap><font face="helvetica"><b>
Show:&nbsp;
<a href="/search.gw?s=prison%2A+AND+escape%2A&c=timely&start=50&showSummary=false
">Headlines</a>
&nbsp;
View by:&nbsp;
<a href="/search.gw?sort=pub&c=timely&s=prison%2A+AND+escape%2A">Publication</a>&nbsp;
or&nbsp;
<a href="/search.gw?c=timely&sort=date&s=prison%2A+AND+escape%2A">Date</a>&nbsp;
</b></font>
</td>
</tr></table>
<br>
<table border=0 cellpadding=2 cellspacing=1 width=120 align=right>
<tr>
<td>
<table border="2" cellspacing="0" cellpadding="3" bordercolor="#330099" bgcolor="#ffffcc" width=120 height=66>
<tr>
<td align="center" width=100%>
<CENTER>
<B>
<FONT FACE="Verdana, Arial, Helvetica, sans-serif">
<FONT SIZE=-1>Beyond.com</FONT>
</FONT>
</B><BR>
<FONT FACE="Verdana, Arial, Helvetica, sans-serif">
<FONT SIZE=-2><a href=http://www.excite.com/relocate/co=20115;http://www.beyond.com/AF20417/search.htm?Query=Query&toggle=name&name_desc=escape>escape</a></FONT>
</FONT><BR>
<FONT FACE="Verdana, Arial, Helvetica, sans-serif">
<FONT SIZE=-2>Buy Software!</FONT>
</FONT>
</CENTER>
</td></tr></table>
</td>
</tr>
<tr>
<td align=center>
<table border="2" cellspacing="0" cellpadding="3" bordercolor="#336600" width=120 height=66>
<tr><td align="center" bgcolor="#ffffcc">
<b><font face="Verdana, Arial, Helvetica, sans-serif" size="-1">Amazon.com</font></b><br>
</td></tr>
<tr><td align="center">
<font face="Verdana, Arial, Helvetica, sans-serif" size="-2">
Search for:<br>
<a href=/relocate/co=xsr_amazon;http://www.amazon.com/exec/obidos/excite-search/search=prison%2A+AND+escape%2A>prison* AND es...</a><br>
</font>
</td></tr></table>
</td>
</tr>
</table>
<font face="helvetica" size=-1>
<b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://insidedenver.com:80/news/0627nulw6.shtml">New laws tweak Colorado lifestyles </A></b>&nbsp;
<SMALL>
(Rocky Mountain News)
</SMALL>
<BR><SMALL>51% </SMALL> &nbsp;&nbsp;Weather. &nbsp;&nbsp;Births.
<BR><i>First found: </i>27 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=17592&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.foxnews.com:80/health/062899/y2hosp.sml">Hospitals Say They're Ready for 2000</A></b>&nbsp;
<SMALL>
(Fox News)
</SMALL>
<BR><SMALL>50% </SMALL>="t") {this.sec=1;pbw(vid,neth,netl,realh,reall);}} if (cm=="rh" && realh=="t") {cm="hi.rmm";if(aol||iebug){window.location=vid+cm;}else{showvid('rm',vid+cm);}} else {if (cm=="rh" && realh!="t") {this.sec=2;pbw(vid,neth,netl,realh,reall);}} if (cm=="rl" && reall=="t") {cm="lo.rmm";if(aol||iebug){window.location=vid+cm;}else{showvid('rm',vid+cm);}} else {if (cm=="rl" && reall!
<BR><i>First found: </i>28 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=13979&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.journalstar.com:80/archives/062999/nat/sto2">Lincoln Journal Star</A></b>&nbsp;
<SMALL>
(Lincoln Journal Star)
</SMALL>
<BR><SMALL>50% </SMALL> About LJS. &nbsp; About LJS.
<BR><i>First found: </i>30 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=6657&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.latimes.com:80/excite/990630/t000058457.html">An Amusing Whodunit With an Endearing Sports Agent </A></b>&nbsp;
<SMALL>
(Los Angeles Times)
</SMALL>
<BR><SMALL>50% </SMALL>Site Index Home Page Archives Discussions Find It Hunter Special Reports ------------- News _Today's Paper _AP The Wire _Traffic _Weather _Commentary _Crossword Calendar Live! Site Index Home Page Archives Discussions Find It Hunter Special Reports ------------- News _Today's Paper _AP The Wire _Traffic _Weather _Commentary _Crossword Calendar Live!
<BR><i>First found: </i>30 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=2522&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.salonmagazine.com:80/news/feature/1999/06/28/china/index.html">Salon News | Why the Cox Report went nowhere</A></b>&nbsp;
<SMALL>
(Salon)
</SMALL>
<BR><SMALL>50% </SMALL> Books in the News. Letters | --> Media | Mothers Who Think | News | People | Technology | Travel &#160;.
<BR><i>First found: </i>29 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=11709&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.usatoday.com:80/life/cyber/tech/ctf487.htm">Hospitals say they're ready for Y2K</A></b>&nbsp;
<SMALL>
(USA Today)
</SMALL>
<BR><SMALL>50% </SMALL> &nbsp;. &nbsp;
<BR><i>First found: </i>28 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=14071&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://cnn.com:80/HEALTH/9906/28/hospitals.y2k.ap/index.html">Hospitals say they're ready for the millennium</A></b>&nbsp;
<SMALL>
(CNN Interactive)
</SMALL>
<BR><SMALL>50% </SMALL> June 17, 1999. June 17, 1999.
<BR><i>First found: </i>28 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=16227&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.govtech.net:80/gtmag/1999/may/prodfocus/prodfocus.shtm">May 99 - Windows 2000: NT Hits the Desktop</A></b>&nbsp;
<SMALL>
(Government Technology)
</SMALL>
<BR><SMALL>36% </SMALL> Y2K Future. Y2K: An Opportunity.
<BR><i>First found: </i>3 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=26318&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.thestar.com:80/thestar/back_issues/ED19990624/news/990624NEW01_FO-WARD24.html">News Story: Hunting Milosevic killing fields - June 24, 1999 </A></b>&nbsp;
<SMALL>
(Toronto Star)
</SMALL>
<BR><SMALL>34% </SMALL> ``Then they found there were too many of them, so they came back with heavy equipment for moving earth.'' Then they found there were too many of them, so they came back with heavy equipment for moving earth.'
<BR><i>First found: </i>24 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=24518&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.smh.com.au:80/news/9906/29/text/obituaries.html">Jim Allen</A></b>&nbsp;
<SMALL>
(Sydney Morning Herald)
</SMALL>
<BR><SMALL>28% </SMALL> He is survived by three daughters and two sons. Widowed several months ago by the death of his wife of 70 years, Flora, Diamond is survived by two children and four grandchildren.
<BR><i>First found: </i>29 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=12716&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.washingtonpost.com:80/wp-srv/WPlate/1999-06/25/046l-062599-idx.html">ON STAGE</A></b>&nbsp;
<SMALL>
(Washington Post)
</SMALL>
<BR><SMALL>01% </SMALL> 301/776-2805. 301/593-6036.
<BR><i>First found: </i>25 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=22609&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> <b><A HREF="http://search.excite.com/relocate/sr=news|ss=prison%2A+AND+escape%2A;http://www.outsidemag.com:80/magazine/0699/9906cook.html">A Long and Brutal Assault</A></b>&nbsp;
<SMALL>
(Outside Magazine)
</SMALL>
<BR><SMALL>01% </SMALL> Cook. Cook!
<BR><i>First found: </i>8 Jun 1999 &nbsp;
<SMALL>
<A HREF="/search.gw?c=timely&qbe=26169&ctag=377A5867&showSummary=true">(Related Articles)</A>
</SMALL>
<P> </font>
<FORM ACTION="/search.gw" METHOD=GET>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH=100%>
<TR><TD>
<INPUT TYPE=hidden NAME=c VALUE=timely>
<INPUT TYPE=hidden NAME=s VALUE="prison* AND escape*">
<INPUT TYPE=hidden NAME=showSummary VALUE=true>
<INPUT TYPE=hidden NAME=start VALUE=50>
<INPUT TYPE=hidden NAME=perPage VALUE=50>
<font face="helvetica" size=-1>
<INPUT TYPE=submit NAME=prev VALUE="Previous Results">
</font>
</TD></TR></TABLE>
</FORM>
<p>
<center>
<form action="http://lifeminders.com/lifeminder30/banner/SignUpDAT.asp"
method="get"> <input type="hidden"
name="MktgSourceCD" value="LLQA58"> <TABLE BORDER=1 CELLPADDING=0
CELLSPACING=0 WIDTH=468 HEIGHT=60><TR><TD>
<table cellpadding="0" cellspacing="0" width="468" height="60"
bgcolor="FF0000" border="0"><tr><td rowspan="2"
valign="top"><img src="http://image.eimg.com/ads/94014_lmhtmlogo.gif" width="96" height="60" border="0"
vspace="0" hspace="0"></td><td height="30"
valign="top" width="372" bgcolor="FF0000"><img src="http://image.eimg.com/ads/94014_contestcopy.gif"
width="370" height="30"
border="0"></td></tr><tr><td valign="top" width="372" height="30"
bgcolor="FF0000"
align="center">&nbsp;&nbsp;<font color="#FFFFFF"><font face="arial,
helvetica, sans-serif" size="-2">Your
E-mail:</font><input type="text" name="Email" size="12" align="top"><font
face="arial, helvetica, sans-serif"
size="-2">&nbsp;&nbsp;Zip Code:</font><input type="text" name="PostalCode"
size="7">&nbsp;&nbsp;&nbsp; <input
type="image" src="http://image.eimg.com/ads/94014_submit.gif" vspace="2" width="50" height="15"
align="bottom"
border="0"></td></tr></table></TD></TR></TABLE><A
HREF="/relocate/ad=94014_htmlad.gif|pk=2920596;http://www.excite.com"></A><CENTER><FONT SIZE=-1
COLOR=#0000FF FACE=arial, geneva><B>Click Here!</B></FONT></CENTER>
</form>
<small><br></small>
<FORM NAME=botbox ACTION=/search.gw>
<TABLE border=0 cellpadding=1 cellspacing=0>
<TR><TD bgcolor=#cccccc>
<TABLE border=0 cellspacing=0 cellpadding=4 width='598' bgcolor=ffffff>
<TR>
<TD align=right bgcolor=ffffff>
<b><font face="helvetica" size=-1>You searched for </font></b>
<input size=30 name=search value="prison* AND escape*">
</td>
<td>
<font face="helvetica" size=-1>
<input type=submit value="Search Again">
</font>
</td>
<TD nowrap>
<font face="helvetica" size=-1>
<b><a href=/relocate/co=help;http://www.excite.com/Info/>
Help</a>&nbsp;&nbsp;<br>
<a href=http://www.excite.com/search_forms/advanced/>
More Search</a>
</b></font>
</TD>
</TR>
</TABLE>
<TABLE border=0 cellspacing=0 cellpadding=4 width=598 bgcolor=ffffff>
<TR>
<td align=right colspan=1>
<font face="helvetica" size=-1>
<b>Search:</b>
</font>
</td>
<td align=left colspan=3>
<font face="helvetica" size=-1>&nbsp;&nbsp;
<input type=radio name=c value=hourly ><img src=http://www.excite.com/pfp/excite/images/lep.gif> Newswires&nbsp;&nbsp;
<input type=radio name=c value=timely checked>Web News&nbsp;&nbsp;
<input type=radio name=c value=hourtime >Both	</font>
</td>
</tr>
</table>
<input type=hidden name=showSummary value=true>
</td></tr>
</table>
</form>
<p>
<font face="helvetica" size=-1>
<nobr><b>More Search Services:</b>&nbsp;
<A HREF=http://www.excite.com/search_forms/advanced?lang=en&c_type=c_country&c_country=web>Advanced Search</A>&nbsp;
<A HREF=http://www.excite.com/search_forms/news>News Search</A>&nbsp;
<A HREF=http://www.excite.com/search_forms/audio_video_search/>Audio/Video Search</A>&nbsp;
<A HREF=http://www.excite.com/voyeur_xt>Search Voyeur</A>&nbsp;
<A HREF=http://www.excite.com/comments/>Feedback</A></nobr><p>
<nobr><b>Services:</b>&nbsp;
<A HREF=http://www.excite.com/info/advertising/>Advertise on Excite</A>&nbsp; <A HREF=/relocate/co=xsr_botnav_bookflights;http://city.net/forms/reservations/>Book Flights</A>&nbsp;
<A HREF=/relocate/co=xsr_botnav_classifieds;http://classifieds2000.com/cgi-cls/display.exe?partner=excite&path=class&template=none&>Classifieds</A>&nbsp;
<A HREF=/relocate/co=7705;http://yellowpages.excite.com>Yellow Pages</A>&nbsp;
<A HREF=http://www.excite.com/reference/>More...</A></nobr><p>
</font>
<font size=-2>
Copyright 1995-1999 Excite Inc. &nbsp; Powered by <A HREF="http://www.sun.com">Sun Microsystems</A> &nbsp;
<A HREF=http://www.excite.com/info/disclaimer/>Disclaimer</A>
</font>
</center>
<IMG SRC="http://ad.preferences.com/ping.ng;spacedesc=TCExciteData_Excite_1x1_RunOfSite_Any&event=KeywordResults&time=1999.06.30.16.03.52" HEIGHT=1 WIDTH=1 BORDER=0>
</body>
</html>
