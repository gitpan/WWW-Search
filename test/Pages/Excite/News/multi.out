http://www.abc.net.au:80/news/newslink/weekly/newsnat-27jun1999-17.htm
http://cnn.com:80/WORLD/europe/9906/26/AM-France-GreatEscape.ap/
http://deseretnews.com:80/dn/view/1,1249,100008585,00.html?
http://www.austin360.com:80/news/2state/1999/06/29escape_escape.html
http://www.austin360.com:80/news/2state/1999/06/29escape_3.html
http://deseretnews.com:80/dn/view/1,1249,100008826,00.html?
http://www.ohio.com:80/bj/news/ohio/docs/002034.htm
http://www.indiaserver.com:80/thehindu/1999/06/28/stories/04282106.htm
http://cnn.com:80/US/9906/29/texas.escaped.killer.ap/
http://www.ohio.com:80/bj/news/ohio/docs/010784.htm
http://cnn.com:80/US/9906/29/carolina.prison.shooting.ap/index.html
http://www.austin360.com:80/news/2state/1999/06/30escape.html
http://www.expressnews.com:80/pantheon/news-bus/state/2902btla.shtml
http://www.expressnews.com:80/pantheon/news-bus/crime-crisis/3002bjha.shtml
http://www.austin360.com:80/news/2state/1999/06/29escape_events.html
http://www.austin360.com:80/news/2state/1999/06/29escape_1.html
http://detnews.com:80/1999/nation/9906/30/06300176.htm
http://www.apbonline.com:80/911/1999/06/29/escape0629_01.html
http://www.suntimes.co.za:80/1999/06/27/news/gauteng/njhbo7.htm
http://www.suntimes.co.za:80/1999/06/27/news/gauteng/njhb05.htm
http://www.journalstar.com:80/archives/062999/nat/sto5
http://www.austin360.com:80/news/2state/1999/06/29escape_2.html
http://www.sltrib.com:80/1999/jun/06281999/utah/4486.htm
http://www.austin360.com:80/news/2state/1999/06/29escapetalkback.html
http://www.apbonline.com:80/911/1999/06/29/breakout0629_01.html
http://www.newsweek.com:80/nw-srv/tnw/today/nm/nm02tu_1.htm
http://www.canoe.com:80/TopStories/killer_jun29.html
http://www.latimes.com:80/excite/990630/t000058618.html
http://www.indiaserver.com:80/thehindu/1999/06/27/stories/0427210y.htm
http://www.theaustralian.com.au:80/state/4219331.htm
http://www.mediainfo.com:80/ephome/news/newshtm/stories/062599n4.htm
http://www.apbonline.com:80/911/1999/06/28/mob0628_01.html
http://www.apbonline.com:80/911/1999/06/28/dna0628_01.html
http://www.bergen.com:80/paterson/snipsnip199906259.htm
http://www.bergen.com:80/bcoast/snipsnip199906259.htm
http://www.apbonline.com:80/911/1999/06/29/email0629_01.html
http://www.spokane.net:80/news-story-body.asp?Date=062499&ID=s598534&cat=
http://www.apbonline.com:80/911/1999/06/29/driver0629_01.html
http://deseretnews.com:80/dn/view/1,1249,100008709,00.html?
http://www.pioneerplanet.com:80/seven-days/1/news/docs/001978.htm
http://www.sltrib.com:80/1999/jun/06241999/nation_w/3589.htm
http://www.sltrib.com:80/1999/jun/06301999/nation_w/4903.htm
http://www.bergen.com:80/bcent/detention199906254.htm
http://www.apbonline.com:80/911/1999/06/28/dapper0628_01.html
http://deseretnews.com:80/dn/view/1,1249,100008589,00.html?
http://www.salonmagazine.com:80/people/col/reit/1999/06/30/paula/index.html
http://www.freep.com:80/voices/columnists/qerubin27.htm
http://www.kcstar.com:80/item/pages/sports.pat,sports/3773a70b.629,.html
http://www.iht.com:80/IHT/TODAY/FRI/IN/missing.2.html
http://insidedenver.com:80/news/0627nulw6.shtml
http://www.foxnews.com:80/health/062899/y2hosp.sml
http://www.journalstar.com:80/archives/062999/nat/sto2
http://www.latimes.com:80/excite/990630/t000058457.html
http://www.salonmagazine.com:80/news/feature/1999/06/28/china/index.html
http://www.usatoday.com:80/life/cyber/tech/ctf487.htm
http://cnn.com:80/HEALTH/9906/28/hospitals.y2k.ap/index.html
http://www.govtech.net:80/gtmag/1999/may/prodfocus/prodfocus.shtm
http://www.thestar.com:80/thestar/back_issues/ED19990624/news/990624NEW01_FO-WARD24.html
http://www.smh.com.au:80/news/9906/29/text/obituaries.html
http://www.washingtonpost.com:80/wp-srv/WPlate/1999-06/25/046l-062599-idx.html
